apps/blog-app/src/app/routes/blog.ts

```diff
@@ -2,6 +2,19 @@ import { Component } from '@angular/core';
 import { RouterLink, RouterOutlet } from '@angular/router';
 import { injectContentFiles } from '@analogjs/content';
 import { NgFor } from '@angular/common';
+import { RouteMeta } from '@analogjs/router';
+
+export const routeMeta: RouteMeta = {
+  title: 'Blog Title',
+  metaTags: [
```

Comment:
@brandonroberts

Thoughts on naming this property `metaTags`, `metatags`, or `meta`? 👀

Reply:
I think `meta` is good. If they ever introduce this property upstream in the router, we may have to adjust things 😅